### PR TITLE
Project

### PR DESCRIPTION
**Pull request by hightechtony**
added mongo db and react folder

### CODE REVIEW

File: react/readme.md [hunk 1]
```diff
@@ -0,0 +1 @@
+## About The Project
```
Comment (sal-strateos) [inline react/readme.md:1]: Let's just call this folder frontend

**Review by sal-strateos — CHANGES_REQUESTED** [review]
Change react name to frontend

**Review by hightechtony — COMMENTED** [review]
looks good

**Comment by hightechtony** [comment]
close

**Pull request closed without merging**